src/worker.ts

```diff
@@ -1,12 +1,21 @@
-import { ping } from './handlers/scheduledEventHandler';
+import { callDiscordNicknameBatchUpdate, ping } from './handlers/scheduledEventHandler';
 import { env } from './types/global.types';
 
 export default {
 	// We need to keep all 3 parameters in this format even if they are not used as as cloudflare workers need them to be present So we are disabling eslint rule of no-unused-vars
 	// for more details read here: https://community.cloudflare.com/t/waituntil-is-not-a-function-when-using-workers-with-modules/375781/4
 	// eslint-disable-next-line no-unused-vars
 	async scheduled(req: Request, env: env, ctx: ExecutionContext) {
-		ctx.waitUntil(ping(env));
+		switch (req.cron) {
+			case '0 */4 * * *':
+				ctx.waitUntil(ping(env));
+				break;
+			case '0 */6 * * *':
+				await callDiscordNicknameBatchUpdate(env);
+				break;
+			default:
+				console.log('Unknown Trigger Value!');
+		}
```

Comment:
we are running cron on every 6 hours, but i think running it every 24 hours because OOO is basedon date.

Reply:
Also be aware that a single account can only have 3 cron triggers so we have bascally used up 2

Reply:
Yes, discussed this with Ankush. He informed to run the job every 6 hours and also use a job for this.